src/main.cpp

```diff
@@ -1,7 +1,8 @@
 #include <iostream>
 
 
-int main() {
+int main()
+{
```

Comment:
May be you forgot to git pull my changes from another pull request. Just revert this change